settings.py

```diff
@@ -41,3 +41,8 @@
 # is not set in a request
 EVPL_SB_PRIORITY = 20000
 EPL_SB_PRIORITY = 10000
+
+#  Time (seconds) to check if an evc has been updated
+#  or flows have been deleted.
```

Comment:
It's great to have these configurable settings. I think the default value should be at least as the worst case as an EVC might take to be installed and also we probably < `DEPLOY_EVCS_INTERVAL`, maybe `DEPLOY_EVCS_INTERVAL // 6` -> 10 secs? 

I think it's good to run some experiments with at least 100+ EVCs and seeing how it'll behave and exercise some updates and flow removals at specific strategic intervals. 

cc'ing @italovalcy and @ajoaoff to contribute to this thread/review

Reply:
yes, I believe that using `DEPLOY_EVCS_INTERVAL` should be good (i.e., if the EVC was recently updated, skip the current consistency check and wait until the next round). WDYT?

Reply:
@italovalcy, I'd expect that an interval value less or slightly less than `DEPLOY_EVCS_INTERVAL` would behave OK, as long as it's also not too small network-convergence-wise.